django_app/Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY django_app/ .
 COPY --from=npm-packages /src/node_modules/ frontend/node_modules/
 COPY --from=poetry-packages /src/venv ./venv
 
-ENV DJANGO_SETTINGS_MODULE='redbox_app.settings'
+ENV DJANGO_SETTINGS_MODULE='redbox_app.settings.py'
```

Comment:
Was this a Daphene related change that can be safely reverted?

Reply:
v good point